test/extensions.js

```diff
@@ -0,0 +1,20 @@
+import { collect } from "./util/collect.js";
+import expect from "expect";
+import { Attribute } from "../lib/productions/attribute.js";
+
+const extensions = {
+  "callback-interface": {
+    extMembers: [[Attribute.parse]],
+  },
+  namespace: {
+    extMembers: [[Attribute.parse, { readonly: false }]],
+  },
```

Comment:
I think it's better to have unit tests, like https://github.com/w3c/webidl2.js/blob/main/test/custom-production.js.